tests/queries/0_stateless/02834_add_sub_date_functions.sql

```diff
@@ -0,0 +1,31 @@
+SELECT addDate('2022-05-07'::Date, INTERVAL 5 MINUTE);
```

Comment:
The test (results) seem susceptible to time zone randomization. Suggest to disable it via a fixed session time zone:

```sql
SET session_timezone = 'UTC';
```